tests/default_vars.sh

```diff
@@ -80,10 +80,8 @@ else
 
 fi
 
-# Re-instantiate COMPILER in case it gets deleted by module purge
-COMPILER=${NEMS_COMPILER:-intel}
 # Longer default walltime for GNU and PGI
-if [[ $COMPILER = gnu ]] || [[ $COMPILER = pgi ]]; then
+if [[ $NEMS_COMPILER = gnu ]] || [[ $NEMS_COMPILER = pgi ]]; then
```

Comment:
Maybe we can remove 'pgi' support.

Reply:
I did that, please check the changes and resolve the conversation if satisfied. Thanks!